### DISCUSSION TITLE
UInt8Array support

### DISCUSSION THREAD

**Discussion by tomdoeslinux** [software-mansion/react-native-reanimated#5643] [Ideas]
When do you plan on adding UInt8Array support

**Comment by piaskowyk** [comment]
Hey, we've already supports UInt8Array in `worklets`. Here is an example - https://github.com/software-mansion/react-native-reanimated/blob/main/app/src/examples/LogExample.tsx#L100

**Comment by tomdoeslinux** [comment]
@piaskowyk Thanks, do you know of an efficient way to update the array instead of having to recreate it ? I"m trying to make a pixel art editor and it should be updated when the user taps on coordinates.

**Comment by gozdecoban** [comment]
Shared values do not support UInt8Array usage, it crashes in the following line.

https://github.com/software-mansion/react-native-reanimated/blob/7aea7e57ca62bbb5c40336967ab9541a4c6c5dbf/packages/react-native-reanimated/Common/cpp/SharedItems/Shareables.cpp#L9

**Reply by tomdoeslinux** [reply]
Moving to Flutter solved my problem.

**Reply by MatiPl01** [reply]
Hi @gozdecoban!
Could you provide a reproducible code example which causes this crash? I am trying to reproduce the issue but it seems to work fine.
